src/Endpoints/DNS.php

```diff
@@ -153,4 +154,30 @@ public function deleteRecord(string $zoneID, string $recordID): bool
 
         return false;
     }
+
+    /**
+     * @param string $cloudflare_zone_id
+     * @param StreamInterface $bind_file_stream
+     * @return \stdClass
+     */
+    public function importRecordsFromBindFile(
+        string $cloudflare_zone_id,
+        StreamInterface $bind_file_stream
+    ): \stdClass {
+        $options = [
+            'multipart' => [
+                [
+                    'name'     => 'file',
+                    'contents' => $bind_file_stream->getContents(),
+                    'filename' => '@bind_config.txt',
+                ]
+            ]
+        ];
+
+        $response = $this->adapter->post('zones/' . $cloudflare_zone_id . '/dns_records/import', $options);
+
+        $this->body = json_decode($response->getBody());
+
+        return $this->body->result;
```

Comment:
I've been playing with the API and if you upload the same file twice, the messages section contains data about why records were not imported, which will be very useful to see, so I'd suggest returning this:

```suggestion
        return (object)[
            'result'   => $this->body->result,
            'messages' => $this->body->messages,
        ];
```